sql/updates/world/master/2023_01_15_00_world.sql

```diff
@@ -42,3 +42,7 @@ UPDATE `pool_members` SET `spawnId`=`spawnId`+@OINCREMENT WHERE `type`=1 AND `sp
 UPDATE `spawn_group` SET `spawnId`=`spawnId`+@OINCREMENT WHERE `spawnType`=1 AND `spawnId` BETWEEN @OGUIDSTART AND @OGUIDEND;
 UPDATE `smart_scripts` SET `entryorguid`=`entryorguid`-@OINCREMENT WHERE `source_type`=1 AND `entryorguid` BETWEEN -@OGUIDEND AND -@OGUIDSTART;
 UPDATE `conditions` SET `SourceEntry`=`SourceEntry`-@OINCREMENT WHERE `SourceTypeOrReferenceId`=22 AND `SourceId`=1 AND `SourceEntry` BETWEEN -@OGUIDEND AND -@OGUIDSTART;
+
+DELETE FROM `spell_script_names` WHERE `ScriptName` IN ('spell_dru_sudden_ambush');
```

Comment:
Thank you very much, its my first commit. maybe you noticed that, thanks for the patience. I will fix all, and do it in db instead of scripts. I was not noticed about that characteristic about spell_proc, maybe the lack of docs as you say.